tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "lib": ["dom", "esnext.asynciterable", "es2017", "es2016", "es2015"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "paths": { "*": ["src/*", "*", "frontend/*", "backend/*"] },
+    "paths": { "automerge": ["*"]},
```

Comment:
This was required because otherwise `src/uuid.js` was overriding imports of `node_modules/uuid` when running tests.